src/anf-preview/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'Azure NetApp File (ANF)' Command Module
+===================================================================
+
+This package is for the Azure NetApp File (ANF)' module.
+i.e. 'az anf'
```

Comment:
Might want to add a little more info- i.e. how to install the extension, basic usage, link to docs for the service, etc..

Reply:
ok. Install and some examples added. We have no docs to link to yet.